string topic
string node_uri
---
string[] subscriber_uris
